pageserver/src/tenant/ephemeral_file.rs

```diff
@@ -221,9 +221,8 @@ pub fn is_ephemeral_file(filename: &str) -> bool {
 
 impl Drop for EphemeralFile {
     fn drop(&mut self) {
-        // drop all pages from page cache
-        let cache = page_cache::get();
-        cache.drop_buffers_for_immutable(self.page_cache_file_id);
+        // There might still be pages in the [`crate::page_cache`] for this file.
```

Comment:
FTR, these are not visited by rustdoc, rustdoc only scans doc comments.

Reply:
Ack, hoping they will be in the future :)